common/jenkins-agents/nodejs18/ocp-config/bc.yml

```diff
@@ -0,0 +1,66 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: jenkins-agent-nodejs18
+parameters:
+- name: ODS_BITBUCKET_PROJECT
+  description: Bitbucket project name.
+  value: opendevstack
+- name: NEXUS_URL
+  required: true
+- name: NEXUS_AUTH
+  required: true
+- name: REPO_BASE
+  required: true
+- name: ODS_IMAGE_TAG
+  required: true
+  value: latest
+- name: ODS_GIT_REF
+  required: true
+- name: JENKINS_AGENT_DOCKERFILE_PATH
+  value: Dockerfile.centos7
```

Comment:
should we default to ubi8 ?

Reply:
I'm wondering why we have those two flavors of the Dockerfile in the first place.

@michaelsauter @braisvq1996 Do we still need both?

Reply:
We have both because centos7 was used for Openshift 3 installation and ubi8 for Openshift 4 -> [Check config](https://github.com/opendevstack/ods-core/blob/master/configuration-sample/ods-core.env.sample#L224-L227)
Better maintain both until we decide to drop Openshift 3 completely.